examples/Cargo.toml

```diff
@@ -11,13 +11,34 @@ description = """
 AA (ERC-4337) bundler examples
 """
 
-[dev-dependencies]
-
+[dependencies]
+alloy-primitives = "0.3"
+alloy-sol-types = "0.3"
 anyhow = "1"
 ethers = { version = "2.0.4" }
+reqwest = { version = "0.11.4", features = ["json"] }
+serde = "1"
+serde_json = "1"
+silius-contracts = { path = "../crates/contracts" }
 silius-primitives = { path = "../crates/primitives" }
+silius-tests = { path = "../tests" }
 tokio = { workspace = true }
 
+[package.metadata.cargo-udeps.ignore]
+normal = ["anyhow", "silius-contracts", "silius-primitives", "silius-tests"]
+
 [[example]]
 name = "user-operation"
 path = "user_operation.rs"
+
+[[example]]
+name = "simple_account_create"
+path = "simple_account/simple_account_create.rs"
+
+[[example]]
+name = "deposit"
```

Comment:
@zsluedem The only thing I would change is the name of example -> simple_account_deposit and later simple_account_tranfer . Otherwise it's good to merge